docs/sample-out/users/2020-11-01-00-00-01.csv

```diff
@@ -1,13 +1,13 @@
-100032893,Schoology,student,larry.mahoney,604927,Larry Mahoney,Larry.Mahoney@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032894,Schoology,student,roland.phillips,604938,Roland Phillips,Roland.Phillips@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032895,Schoology,student,stephen.caldwell,604969,Stephen Caldwell,Stephen.Caldwell@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032896,Schoology,student,olivia.hardy,604974,Olivia Doris Hardy,Olivia.Hardy@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032897,Schoology,student,micheal.turner,605015,Micheal Turner,Micheal.Turner@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032898,Schoology,teacher,kelley.christian,207270,Kelley Heidi Christian,Kelley.Christian@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032899,Schoology,teacher,sara.preston,207268,Sara Stacy Preston,Sara.Preston@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-100032900,Schoology,teacher,sara.preston.2,207268,Sara Stacy Preston,Sara.Preston@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-1,Schoology,principal,delete.me,,This person will be un-deleted from the next file,principal@studentgps.org,2020-11-01 00:00:01,2020-11-01 00:00:01,,
\ No newline at end of file
+100032890,BestLMS,student,mary.archer,604863,Mary Archer,Mary.Archer@studentgps.org,2020-08-20 12:34:56,2020-09-18 12:34:00,,
+100032891,BestLMS,student,kyle.hughes,604874,Kyle Hughes,Kyle.Hughes@studentgps.org,2020-08-20 12:34:56,2020-09-18 12:34:00,,
+100032892,BestLMS,student,peter.nash,604918,Peter Ivan Nash,Peter.Nash@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032893,BestLMS,student,larry.mahoney,604927,Larry Mahoney,Larry.Mahoney@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032894,BestLMS,student,roland.phillips,604938,Roland Phillips,Roland.Phillips@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032895,BestLMS,student,stephen.caldwell,604969,Stephen Caldwell,Stephen.Caldwell@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032896,BestLMS,student,olivia.hardy,604974,Olivia Doris Hardy,Olivia.Hardy@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032897,BestLMS,student,micheal.turner,605015,Micheal Turner,Micheal.Turner@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032898,BestLMS,teacher,kelley.christian,207270,Kelley Heidi Christian,Kelley.Christian@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032899,BestLMS,teacher,sara.preston,207268,Sara Stacy Preston,Sara.Preston@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+100032900,BestLMS,teacher,sara.preston.2,207268,Sara Stacy Preston,Sara.Preston@studentgps.org,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+1,BestLMS,principal,delete.me,,This person will be un-deleted from the next file,principal@studentgps.org,2020-11-01 00:00:01,2020-11-01 00:00:01,,
```

Comment:
Adding this record because we have this user identifier in the system activities file on four records. Those records are not uploading because this user is missing.

```suggestion
1,BestLMS,principal,delete.me,,This person will be un-deleted from the next file,principal@studentgps.org,2020-11-01 00:00:01,2020-11-01 00:00:01,,
99588912,BestLMS,principal,Juana.Roe,,Juana Roe,juana.roe@studentgps.org,2021-01-01 00:00:01,2021-01-01 00:00:01,,

```